packages/desktop-client/src/components/reports/spreadsheets/spending-spreadsheet.ts

```diff
@@ -167,6 +200,7 @@ export function createSpendingSpreadsheet({
         average: integerToAmount(averageSum) / monthCount,
         thisMonth: dayData[3].cumulative,
         lastMonth: dayData[2].cumulative,
```

Comment:
These need to be adjusted. The "Spent MTD:" in the top right corner is showing last month instead of this month and the "Spent Last MTD:" is showing 2 months ago.